### PR TITLE
Update proxy container installation page

### PR DESCRIPTION
**Pull request by nodeg**
# Description

- More containerized proxy installation fixes.
  - SLE Micro needs 2 reboots in order to be fully onboarded
  - I moved updating the minion to its own procedure
  - I did test this with a fresh SLEM 5.5 Minion. See https://github.com/SUSE/spacewalk/issues/24184
- typo fixes

Furthermore, I would move `mgrxpy` completely to the proxy setup page if we would like to have 2 pages for the proxy Preparation/Installation/Deployment.

Preview with e.g.:

```bash
uyuni-docs-helper -o . -c obs-packages-suma-en -g https://github.com/nodeg/uyuni-docs.git -r container-proxy -p suma
```

# Target branches

master

Backport targets (edit as needed):

no.

# Links

### CODE REVIEW

**Review by jcayouette — APPROVED** [review]

**Comment by jcayouette** [comment]
This is great, thank you!

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-container-setup.adoc [hunk 1]
```diff
@@ -107,11 +107,13 @@ The following procedure assumes you have added your Organization Credentials to
 . Fill in the fields for your Proxy host.
 . Select the Activation key created in the previous step from the dropdown.
 . Click btn:[+ Bootstrap].
-. Wait for the Bootstrap process to complete successfully. Check the menu:Systems[] list. The {salt} minion will be listed once complete.
-. Seleect the host from the menu:Systems[] list and apply all patches to update it.
+. Wait for the Bootstrap process to complete successfully. Check the menu:Salt[] menu and confirm the {salt} minion key is listed and accepted.
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-container-setup.adoc:110]:
```suggestion
. Wait for the Bootstrap process to complete successfully.
  Check the menu:Salt[] menu and confirm the {salt} minion key is listed and accepted.
```

**Review by keichwa — APPROVED** [review]
Thanks!  In general, we probably still want every sentence to start on a line of its own (mostly to help the segmentation process with creating proper template chunks).

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-container-setup.adoc (hunk 1 above)
Comment (nodeg) [reply modules/installation-and-upgrade/pages/container-deployment/suma/proxy-container-setup.adoc]: In the rendered version it is still in the same line, though.
image

**Pull request merged**